docs/BR.md

```diff
@@ -751,7 +751,14 @@ The Random Value SHALL remain valid for use in a confirming response for no more
 
 **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.
 
-Effective November 1, 2024, Subscriber Certificates MUST NOT be issued that 1) rely on WHOIS to identify Domain Contact information or 2) reuse validation data where WHOIS was used to identify the Domain Contact.  
+Effective January 15, 2025, when issuing Subscriber Certificates…
+- The CA MUST NOT rely on Domain Contact information obtained using an HTTPS website, regardless of whether previously obtained information is within the allowed reuse period.
+- The CA MUST NOT rely on Domain Contact information obtained using the WHOIS protocol (RFC 3912) or the Registry Data Access Protocol (RFC 7482) if the requested Domain Name contains a ccTLD, regardless of whether previously obtained information is within the allowed reuse period.
+- When obtaining Domain Contact information using the WHOIS protocol, the CA MUST query IANA's WHOIS server and follow referrals to the appropriate gTLD WHOIS server.
+- When obtaining Domain Contact information using the Registry Data Access Protocol, the CA MUST utilize IANA's bootstrap file to identify and query the correct RDAP server for the domain.
+- The CA SHOULD NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA to ensure that it relies upon up-to-date and accurate information.
```

Comment:
I propose:

> The CA MUST NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA that is more than 48 hours old,  to ensure that it relies upon up-to-date and accurate information

Rationale: 48 hours is the TTL on the NS records in the root zones, so this provides parity with DNS.  I don't think CAs should have discretion to ignore this.

(Ditto for the other sections where this language is used.)

Reply:
Thanks for your review and feedback, Andrew! Your suggestion and justification are reasonable to me.

I'll leave this comment open to help facilitate additional discussion.

Reply:
Given no further comments, I actioned this comment in [b740449](https://github.com/ryancdickson/staging/commit/b740449d80c71ec4f3cefab2e68eb86ba2ccc924).